addons/web/static/src/views/fields/image/image_field.js

```diff
@@ -105,8 +97,6 @@ export class ImageField extends Component {
     }
     onFileUploaded(info) {
         this.state.isValid = true;
-        // Invalidate the `rawCacheKey`.
-        this.rawCacheKey = null;
```

Comment:
what if there's no onchange that modifies the last_update field when a new image is uploaded? How is the new image correctly displayed in this case?